src/assets/i18n/en.json5

```diff
@@ -4327,6 +4335,10 @@
 
   "relationships.OrgUnit.isPublicationOfAuthor.Publication": "Authored Publications",
 
+  "relationships.OrgUnit.isPublicationOfContributor.Publication": "Publications",
```

Comment:
Same as above, this should likely be something like "Publications (contributor to)".